library/Erfurt/Store/Adapter/EfZendDb.php

```diff
@@ -330,7 +345,7 @@ protected function _getNormalizedErrorCode()
                     return 1000;
             }
         } else {
-            return -1;
+            return Erfurt_Exception::DATABASE_ERROR;
```

Comment:
Why do you replace the return value in this case with a constant, but not three lines above?